.buildkite/pipeline.trigger.integration.tests.sh

```diff
@@ -22,6 +22,8 @@ STACK_COMMAND_TESTS=(
     test-stack-command-8x
     test-stack-command-9x
     test-stack-command-with-apm-server
+    test-stack-command-with-basic-license
+    test-stack-command-with-self-monitor
```

Comment:
This was added as a target in the Makefile but it was missing in the script that triggers the steps.